client/post-editor/post-editor.jsx

```diff
@@ -340,56 +386,68 @@ export const PostEditor = React.createClass( {
 									type={ this.props.type }
 								/>
 							</div>
-							<FeaturedImage
-								site={ site }
-								post={ this.state.post }
-								maxWidth={ 1462 } />
-							<div className="post-editor__header">
-								<EditorTitle
-									onChange={ this.onEditorTitleChange }
-									tabIndex={ 1 } />
-								{ this.state.post && isPage && site
-									? <EditorPageSlug
-										path={ this.state.post.URL && ( this.state.post.URL !== siteURL )
-											? utils.getPagePath( this.state.post )
-											: siteURL
-										}
-										/>
-									: null
-								}
-								<SegmentedControl className="post-editor__switch-mode" compact={ true }>
-									<SegmentedControlItem
-										selected={ mode === 'tinymce' }
-										onClick={ this.switchEditorVisualMode }
-										title={ this.props.translate( 'Edit with a visual editor' ) }>
-										{ this.props.translate( 'Visual', { context: 'Editor writing mode' } ) }
-									</SegmentedControlItem>
-									<SegmentedControlItem
-										selected={ mode === 'html' }
-										onClick={ this.switchEditorHtmlMode }
-										title={ this.props.translate( 'Edit the raw HTML code' ) }>
-										HTML
-									</SegmentedControlItem>
-								</SegmentedControl>
+							<div className={ classNames(
+								'post-editor__inner-content',
+								{ 'is-shown': this.state.nestedSidebar === NESTED_SIDEBAR_NONE }
+							) }>
+								<FeaturedImage
+									site={ site }
+									post={ this.state.post }
+									maxWidth={ 1462 } />
```

Comment:
Indeed, this should be better explained. I traced it back to `10818-gh-calypso-pre-oss` 😅 . As the saying goes, patches welcome :trollface: . (In any case, doesn't belong in this PR; not to mention that the way GitHub renders the diff, it'd be very hard to spot that change in here.)